packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -374,15 +374,27 @@ export class StripeHelper extends StripeHelperBase {
     email: string,
     displayName: string,
     idempotencyKey: string,
-    ipAddress?: string
+    countryCode?: string,
+    postalCode?: string
   ): Promise<Stripe.Customer> {
+    const shipping =
+      countryCode && postalCode
+        ? {
+            name: `Maxmind GeoIP2 ${new Date()}`,
```

Comment:
Is this going to show up on the invoices that Stripe generates?

Reply:
I didn't see it showing up there, but out of an abundance of caution might be good to change this?

Reply:
We could store it on metadata is we want to remember it.